.github/workflows/release.yml

```diff
@@ -0,0 +1,24 @@
+name: Build and Release to npm
+on:
+  push:
+    branches:
+      - 'semantic-release'
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    env:
+      NPM_REGISTRY: npm.pkg.github.com/abernier
+      NPM_REGISTRY_URL: https://npm.pkg.github.com/abernier
+      NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
@abernier 
Would it be possible to change `GITHUB_TOKEN` to `GH_TOKEN`?
They don't accept `GITHUB_***`.

![image](https://user-images.githubusercontent.com/212837/218327832-23c50291-f10d-400e-a239-c0fe99b9a9cf.png)

Reply:
from what I know, `GITHUB_TOKEN` is automatically provided by Github at start of each workflow run, so there is no need to set it up

cf. https://docs.github.com/en/actions/security-guides/automatic-token-authentication#about-the-github_token-secret

Reply:
Also I didn't managed to get this action working with my `abernier`'s Github npm registry yet: https://github.com/abernier/camera-controls/actions

I'll have other tries tomorrow bit feel free to test everything you want in the meantime ;)